src/components/tutorialComponent/index.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint react/no-array-index-key: 0 */
+// NOTE: We use the index for the the array.map function simply because
+// we don't modify the array afterwords, so the index will always be correct
+import React, { useGlobal } from "reactn";
+import styled, { AnyStyledComponent } from "styled-components";
+import { useStaticQuery, graphql } from "gatsby";
+import { Tutorial, Content } from "../interfaces";
+
+interface TutorialProps {
+  display: string;
+}
+
+const TutorialBG: AnyStyledComponent = styled.div`
+  background-color: #fff;
+  border-radius: 5px;
+  display: ${(props: TutorialProps): string => props.display};
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  height: 90%;
+  width: 70%;
+
+  h1 {
+    font-size: 32px;
+  }
+`;
+
+const Title: AnyStyledComponent = styled.h1`
+  text-align: left;
+  font-size: 32px;
+  font-weight: normal;
+  margin: 20px 0 0 0;
+  width: 80%;
+`;
+
+const StepContentWrapper: AnyStyledComponent = styled.div`
+  height: 100%;
+  max-height: 55vh;
+  overflow-y: auto;
+`;
+
+const ContentWrapper: AnyStyledComponent = styled.div`
+  width: 80%;
+  height: 100%;
+  font-size: 16px;
+`;
+
+interface ButtonProps {
+  next: boolean;
+}
+
+const Button: AnyStyledComponent = styled.button`
+  height: 2.5rem;
+  width: 6rem;
+
+  background-color: ${(props: ButtonProps): string =>
+    props.next ? "#87C5FF" : "#C5C5C5"};
+  border: none;
+  border-radius: 5px;
+
+  font-size: 18px;
+`;
+
+const CodeBlock: AnyStyledComponent = styled.div`
+  padding: 0.5rem 1rem;
+  font-family: "Source Code Pro";
+  background-color: #e8e8e8;
+  border-radius: 5px;
+
+  span {
+    white-space: pre;
+  }
+`;
+
+const ButtonGroup: AnyStyledComponent = styled.div`
+  width: 80%;
+  margin: 2rem 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const TutorialDisplay: React.FC = (): JSX.Element => {
+  const [tutorialStep, setTutorialStep] = useGlobal("tutorialStep");
+  const [tutorialName] = useGlobal("tutorialName");
+  const [tutorialDisplay] = useGlobal("tutorialDisplay");
+
+  let data = useStaticQuery(graphql`
+    query {
+      allExampleGqlJson {
+        nodes {
+          tutorial_title
+          instructions {
+            step
+            title
+            type
+            content {
+              type
+              value
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  // We destructure the data since this query returns an array, and when
+  // we use the GraphQL filter it'll end up being an array of size 1. Otherwise
+  // it just picks the first element
+  [data] = data.allExampleGqlJson.nodes.filter(
```

Comment:
While I was the one who suggested you to use `filter()`, I believe now that `find()` might actually be a better function to use in case someone accidentally uploads a tutorial for the same name, because `filter()` is able to return more than one element which may cause further bugs down the line.